README.md

```diff
@@ -31,42 +24,27 @@ Have a bug or feature request? [File a bug on Launchpad][fileabug].
 Want to get involved in Mixxx development? Assign yourself a bug from the [easy
 bug list][easybugs] and get started!
 
-## Compiling on Linux
-    $ mkdir build
-    $ cd build
-    $ cmake ..
-    $ cmake --build .
-Please see our helpful guide on the [wiki] for more information: [Compiling on Linux]
+## Building Mixxxx
```

Comment:
Is Mixxx spelled with four 'x' deliberately here or is this a typo?

Reply:
typo, thanks for pointing that out.